content/client-server-api/modules/push.md

```diff
@@ -293,6 +293,94 @@ For an example of this, see the default rule
 [`.m.rule.tombstone`](#mruletombstone) below.
 {{% /boxes/warning %}}
 
+**`event_property_is`**
+
+This is an exact value match on a property of the event. Parameters:
+
+-   `key`: The [dot-separated path of the property](/appendices#dot-separated property paths)
+    of the event to match, e.g. `content.body`.
+
+-   `value`: The value to match against.
+
+The match is performed exactly and only supports strings, integers, booleans, and
+`null`.
+
+If the property specified by `key` is completely absent from the event, or does
+not have a string, integer, boolean, or `null` value, then the condition will not
+match.
+
+{{% boxes/note %}}
+For example, if `key` is `content.m.federate`, and `value` is `true`, then
```

Comment:
This isn't escaped properly:

```suggestion
For example, if `key` is `content.m\\.federate`, and `value` is `true`, then
```

I wanted to use an example of a real field -- if there are better ones than this, please let me know of ideas!

Reply:
> Why don't we escape it properly here, since we're also introducing [MSC3873](https://github.com/matrix-org/matrix-spec-proposals/issues/3873)

Because I forgot to do it in the PR so was making a note to myself -- I was curious if a reviewer had a better property to check against.